src/Microsoft.Data/PrimitiveDataFrameColumnContainer.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public partial class PrimitiveDataFrameColumnContainer<T>
+        where T : struct
+    {
+        public IList<DataFrameBuffer<T>> Buffers = new List<DataFrameBuffer<T>>();
+        public PrimitiveDataFrameColumnContainer(T[] values)
+        {
+            values = values ?? throw new ArgumentNullException(nameof(values));
+            long ii = 0;
```

Comment:
The variable can be in the `for` loop on line 30.

BTW - standard convention is just to use `i` in for loops (or a full named variable).  See [https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/for#structure-of-the-for-statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/for#structure-of-the-for-statement)

```C#
for (int i = 0; i < 5; i++)
{
    Console.WriteLine(i);
}
```